config/values/routes_into_teaching.yml

```diff
@@ -0,0 +1,130 @@
+routes:
```

Comment:
@spencerldixon we've used the `/values` folder just for financial values - would it make more sense to have `/config/routes_into_teaching` as the path?